importer/DkBygningDa.py

```diff
@@ -59,6 +56,11 @@ def set_inception(self):
                 self.add_statement(
                     "inception", {"time_value": {"year": inception}})
 
+    def set_monuments_all_id(self):
+        """ Map monuments_all ID to fields in this table. """
+        self.monuments_all_id = "{}-{}-{}".format(
```

Comment:
I believe you can add `:s` inside each `{}` to remove the need for `str`-casting. Check before trusting me though =)